### PR TITLE
javascript/collatz-conjecture: initial mentor notes

### PR DESCRIPTION
**Pull request by SleeplessByte**

### CODE REVIEW

File: tracks/javascript/exercises/collatz-conjecture/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+### Reasonable solution
+
+The collatz conjecture has multiple approaches when it comes to implementation,
+and both a recursive as an iterative implementation can be idiomatic.
```
Comment (tejasbubane) [inline tracks/javascript/exercises/collatz-conjecture/mentoring.md]: recursive `and` an iterative ?

Comment (SleeplessByte) [reply tracks/javascript/exercises/collatz-conjecture/mentoring.md]: I wanted to say "as well as", but I like yours better.

Comment (SleeplessByte) [inline tracks/javascript/exercises/collatz-conjecture/mentoring.md]:
```suggestion
both a recursive and an iterative implementation can be idiomatic.
```

**Comment by SleeplessByte** [comment]
@ihid can you restart this one? (CI)

**Comment by tejasbubane** [comment]
@SleeplessByte Please rebase this one on master.

**Comment by hayashi-ay** [comment]
Why hasn't this PR still merged?

**Comment by tejasbubane** [comment]
I had missed this. Thanks for reminding @hayashi-ay 👍

**Pull request merged**